ansible_runner/interface.py

```diff
@@ -91,7 +93,15 @@ def init_runner(**kwargs):
     event_callback_handler = kwargs.pop('event_handler', None)
     status_callback_handler = kwargs.pop('status_handler', None)
     artifacts_handler = kwargs.pop('artifacts_handler', None)
-    cancel_callback = kwargs.pop('cancel_callback', None)
+    if kwargs.get('cancel_callback'):
+        cancel_callback = kwargs.pop('cancel_callback')
```

Comment:
For easier reading, just `pop` the value and store it.

```suggestion
    cancel_callback = kwargs.pop('cancel_callback')
    if cancel_callback:
```

Reply:
Updated

Reply:
✅